### PR TITLE
Update use.json

### PR DESCRIPTION
**Pull request by Zearin**
Edited `notes` to use terser language (and note the version that fixed the bug at the beginning).

----

A checklist to help your pull request get merged faster:
- [x] Summarize your changes

----

(Other checklist items omitted, as they are not applicable.)

### CODE REVIEW

File: svg/elements/use.json [hunk 1]
```diff
@@ -16,11 +16,11 @@
             },
             "firefox": {
               "version_added": "4",
-              "notes": "For years, Firefox has suffered from a bug whereby it doesn't completely follow the <code>use</code> cascading rules (see <a href='https://bugzil.la/265894'>bug 265894</a>). A fix is <a href='https://stackoverflow.com/questions/27866893/svg-fill-not-being-applied-in-firefox/27872310#27872310'>documented by Amelia Bellamy-Royds on StackOverflow</a>. The good news is that this is finally fixed as of Firefox 56."
+              "notes": "Firefox versions prior to 56 suffered from a bug where they didn't completely follow the <code>use</code> cascading rules (see <a href='https://bugzil.la/265894'>bug 265894</a>). A fix is <a href='https://stackoverflow.com/questions/27866893/svg-fill-not-being-applied-in-firefox/27872310#27872310'>documented by Amelia Bellamy-Royds on StackOverflow</a>."
```
Comment (ddbeck) [inline svg/elements/use.json:19]:
```suggestion
              "notes": "Before version 56, Firefox does not completely follow the <code>use</code> cascading rules (see <a href='https://bugzil.la/265894'>bug 265894</a>). A fix is <a href='https://stackoverflow.com/questions/27866893/svg-fill-not-being-applied-in-firefox/27872310#27872310'>documented by Amelia Bellamy-Royds on StackOverflow</a>."
```

File: svg/elements/use.json [hunk 2]
```diff
@@ -16,11 +16,11 @@
...
             },
             "firefox_android": {
               "version_added": "4",
-              "notes": "For years, Firefox has suffered from a bug whereby it doesn't completely follow the <code>use</code> cascading rules (see <a href='https://bugzil.la/265894'>bug 265894</a>). A fix is <a href='https://stackoverflow.com/questions/27866893/svg-fill-not-being-applied-in-firefox/27872310#27872310'>documented by Amelia Bellamy-Royds on StackOverflow</a>. The good news is that this is finally fixed as of Firefox 56."
+              "notes": "Firefox versions prior to 56 suffered from a bug where they didn't completely follow the <code>use</code> cascading rules (see <a href='https://bugzil.la/265894'>bug 265894</a>). A fix is <a href='https://stackoverflow.com/questions/27866893/svg-fill-not-being-applied-in-firefox/27872310#27872310'>documented by Amelia Bellamy-Royds on StackOverflow</a>."
```
Comment (ddbeck) [inline svg/elements/use.json:23]:
```suggestion
              "notes": "Before version 56, Firefox does not completely follow the <code>use</code> cascading rules (see <a href='https://bugzil.la/265894'>bug 265894</a>). A fix is <a href='https://stackoverflow.com/questions/27866893/svg-fill-not-being-applied-in-firefox/27872310#27872310'>documented by Amelia Bellamy-Royds on StackOverflow</a>."
```

**Review by ddbeck — CHANGES_REQUESTED** [review]
Hi @Zearin and thank your for this PR. I see that it's your first to BCD—welcome to the project!

This is a really good edit. I put in a couple suggestions below to tighten up the language even more.

**Comment by Zearin** [comment]
Thanks, @ddbeck !  Suggestions committed.

**Review by ddbeck — APPROVED** [review]
Thank you! 🎉

**Pull request merged**